RHIDP-6571 - remove configuring the base URL for Operator-based (#1096)
redhat-developer/red-hat-developers-documentation-rhdh: modules/configuring/proc-provisioning-your-custom-configuration.adoc

. Author your custom `{my-app-config-file}` file.
This is the main {product-short} configuration file.

> ** To prepare a deployment with the {product} Operator, you can start with an empty file.

Author: ```suggestion
** To prepare a deployment with the {product} Operator on OpenShift, you can start with an empty file.
```

> ** To prepare a deployment with the {product} Operator on OpenShift, you can start with an empty file.

Author: ```suggestion
** To prepare a deployment with the {product} Operator on OpenShift, you can start with an empty file.
Creating a custom file, even an empty one, is required to avoid the {product} Operator to revert user edits during upgrades.
```

. Author your custom `{my-app-config-file}` file.
This is the main {product-short} configuration file.

** To prepare a deployment with the {product} Operator on OpenShift, you can start with an empty file.
Creating a custom file, even an empty one, is required to avoid the {product} Operator to revert user edits during upgrades.

** To prepare a deployment with the {product} Helm chart, or on Kubernetes, add the `baseUrl` field in your `{my-app-config-file}` file to ensure proper functionality of {product-short}.

> Specify the `baseUrl` in both the `app` and `backend` sections to avoid errors during initialization.

Reviewer: Can you also mention the `backend.cors.origin` field? This is equally important as well, as it might otherwise have an impact on using RHDH from the browser.

Author: Done

> ** To prepare a deployment with the {product} Helm chart, or on Kubernetes, add the `baseUrl` field in your `{my-app-config-file}` file to ensure proper functionality of {product-short}.
> Specify the `baseUrl` in both the `app` and `backend` sections to avoid errors during initialization.

Author: ```suggestion
** To prepare a deployment with the {product} Helm chart, or on Kubernetes, enter the {product-short} base URL in the relevant fields your `{my-app-config-file}` file to ensure proper functionality of {product-short}.
The relevant fields are `baseUrl` in the `app` and `backend` section, and `origin` in the `backernd.cors` subsection:
```

> The relevant fields are `baseUrl` in the `app` and `backend` section, and `origin` in the `backernd.cors` subsection:

Author: ```suggestion
The base URL is be what the user see in their browser when accessing {product-short}.
The relevant fields are `baseUrl` in the `app` and `backend` section, and `origin` in the `backernd.cors` subsection:
```

. Author your custom `{my-app-config-file}` file.

> This is the main {product-short} configuration file.

Author: ```suggestion
This is the main {product-short} configuration file.
You need a custom `{my-app-config-file}` file to avoid the {product-short} installer to revert user edits during upgrades.
When your custom `{my-app-config-file}` file is empty, {product-short} is using default values.
```

> ** To prepare a deployment with the {product} Operator on OpenShift, you can start with an empty file.

Reviewer 2: replace OpenShift with the OCP product name attribute -- we are advised to use _OpenShift Container Platform_ (per the attribute) and not just _OpenShift_ by itself.

Author: ```suggestion
** To prepare a deployment with the {product} Operator on {ocp-short}, you can start with an empty file.
```

> Creating a custom file, even an empty one, is required to avoid the {product} Operator to revert user edits during upgrades.

Reviewer 2: Format this as a NOTE admonition or move it to the short description section the module to avoid presenting a loose, random fact between steps

Author: Moved above.

```suggestion
```

> ** To prepare a deployment with the {product} Helm chart, or on Kubernetes, enter the {product-short} base URL in the relevant fields your `{my-app-config-file}` file to ensure proper functionality of {product-short}.

Reviewer 3: ...in the relevant fields **in** your `{my-app-config-file}` file to ensure proper functionality of {product-short}.

Author: ```suggestion
** To prepare a deployment with the {product} Helm chart, or on Kubernetes, enter the {product-short} base URL in the relevant fields in your `{my-app-config-file}` file to ensure proper functionality of {product-short}.
```

> The base URL is be what the user see in their browser when accessing {product-short}.

Reviewer 3: ```suggestion
The base URL is what the user sees in their browser when accessing {product-short}.
```

> The base URL is be what the user see in their browser when accessing {product-short}.

Reviewer 2: Since the user is the reader, consider replacing _the user_ and _their_ with _you_ and _your_. This approach also creates consistency with how most other RHDH docs address the reader.

Author: At this place the reader is the OpenShift user creating the RHDH instance, and not the RHDH user.

> The relevant fields are `baseUrl` in the `app` and `backend` section, and `origin` in the `backend.cors` subsection:

Reviewer 2: ```suggestion
The relevant fields are `baseUrl` in the `app` and `backend` sections, and `origin` in the `backend.cors` subsection:
```
2 separate sections

. Author your custom `{my-app-config-file}` file.
This is the main {product-short} configuration file.
You need a custom `{my-app-config-file}` file to avoid the {product-short} installer to revert user edits during upgrades.
When your custom `{my-app-config-file}` file is empty, {product-short} is using default values.

** To prepare a deployment with the {product} Operator on {ocp-short}, you can start with an empty file.

** To prepare a deployment with the {product} Helm chart, or on Kubernetes, enter the {product-short} base URL in the relevant fields in your `{my-app-config-file}` file to ensure proper functionality of {product-short}.

> The base URL is what the user sees in their browser when accessing {product-short}.

Author: ```suggestion
The base URL is what a {product-short} user sees in their browser when accessing {product-short}.
```

    origin: {my-product-url}
----
====

> ** Optionally, enter your configuration such as:

Reviewer 2: CCS recommends writing lead-in sentences as complete sentence. Doing so might also clarify what we are trying to communicate. Are we trying to say something like _Optionally, use one of the following links to enter your configuration:_ 

But you'll be able to write a better alternative sentence since you understand this better.
